### PR TITLE
Add freeze and cleanup entities

### PR DESCRIPTION
**Pull request by Bonyoze**

### CODE REVIEW

File: lua/warden/warden.lua [hunk 1]
```diff
@@ -240,6 +243,46 @@ if SERVER then
...
+		end
+	end
+
+	function Warden.CleanupDisconnected()
+		for steamid, _ in pairs(Warden.Players) do
+			if Warden.PlayerIsDisconnected(steamid) then
+				Warden.CleanupEntities(steamid)
+			end
+		end
+	end
```
Comment (anormaltwig) [inline lua/warden/warden.lua:288]: The two functions above here don't seem to be used anywhere, what is the purpose of them?

Comment (anormaltwig) [reply lua/warden/warden.lua:288]: Actually I guess there's other unused functions too so it doesn't really matter.

File: lua/warden/convars.lua [hunk 2]
```diff
@@ -0,0 +1,8 @@
+if SERVER then
+	CreateConVar("warden_freeze_disconnect", 1, nil, "Freeze owned entities on player disconnect", 0, 1)
+	CreateConVar("warden_cleanup_disconnect", 1, nil, "Cleanup owned entities on player disconnect", 0, 1)
+	CreateConVar("warden_cleanup_time", 600, nil, "Time in seconds until cleanup after player disconnect", 0)
```
Comment (ANormalTwig) [inline lua/warden/convars.lua:4]: These need to have FCVAR_ARCHIVE as the third argument

Comment (Bonyoze) [reply lua/warden/convars.lua:4]: FCVAR_ARCHIVE is client only
image

**Pull request merged**